check50/__main__.py

```diff
@@ -118,20 +122,19 @@ def setup_logging(level):
     level 'info' logs all git commands run to stderr
     level 'debug' logs all git commands and their output to stderr
     """
-    # No verbosity level set, don't log anything
-    if not level:
-        return
 
-    lib50_logger = logging.getLogger("lib50")
+    for logger in (logging.getLogger("lib50"), LOGGER):
+        # Set verbosity level on the lib50 logger
+        logger.setLevel(level.upper())
 
-    # Set verbosity level on the lib50 logger
-    lib50_logger.setLevel(getattr(logging, level.upper()))
+        handler = logging.StreamHandler(sys.stderr)
+        handler.setFormatter(ColoredFormatter("(%(levelname)s) %(message)s"))
 
-    # Direct all logs to sys.stderr
-    lib50_logger.addHandler(logging.StreamHandler(sys.stderr))
+        # Direct all logs to sys.stderr
+        logger.addHandler(handler)
 
-    # Don't animate the progressbar
-    lib50.ProgressBar.DISABLED = True
+    # Don't animate the progressbar if LogLevel is either info or debug
+    lib50.ProgressBar.DISABLED = logger.level < LogLevel.WARNING
```

Comment:
Does that still work as expected with `WARNING` now set to `auto()`, which I think is `2` in this case?

Reply:
Whoops good catch! Using `logging.INFO`, `logging.ERROR` etc. now instead of `enum.auto()`